lib/fizzbuzz.rb

```diff
@@ -2,10 +2,19 @@ class SuperFizzBuzz
 
   def run(input)
 
-    #Implement your code here
-
+    if input.modulo(3).zero?
```

Comment:
Just as a matter of style, I think [guard clauses](https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals) make this simpler for other developers to read:

```ruby
return "FizzBuzz" if input.modulo(15).zero?
```